packages/datadog-plugin-mysql/src/index.js

```diff
@@ -23,14 +23,15 @@ class MySQLPlugin extends DatabasePlugin {
       }
     })
   }
-}
 
-function getServiceName (config, dbConfig) {
-  if (typeof config.service === 'function') {
-    return config.service(dbConfig)
-  }
+  getServiceName (dbConfig) {
```

Comment:
It looks like it is the same implementation in mysql, oracledb and pg, can't we move this method to parent `DatabasePlugin` class?

Reply:
I really wanted to generalize it but the service name calculation was so different across implementations.